discovery-infra/tests/base_test.py

```diff
@@ -106,13 +106,17 @@ def wait_until_hosts_are_registered(
     def wait_until_hosts_are_discovered(
         cluster_id,
         api_client,
-        nodes_count=env_variables['num_masters']+env_variables['num_workers']
+        nodes_count=env_variables['num_masters']+env_variables['num_workers'],
```

Comment:
base class methods moved to cluster, we are deprecating most of the methods here